lib/main_common.pm

```diff
@@ -2426,4 +2426,29 @@ sub load_transactional_role_tests {
     loadtest 'transactional_system/rebootmgr';
 }
 
+# Tests to validate partitioning with LVM, both encrypted and not encrypted.
+# Also covered a case while installing on a system with a cryptlvm volume
+# present (e.g. previous clean installation using cryptlvm).
+# If 'ENCRYPT_FORCE_RECOMPUTE' variable is specified, call the test that
+# reconfigures lvm partition, otherwise partition suggestion based on existing
+# cryptlvm should compute encrypted system (currently it refers to bsc#993247).
+sub load_lvm_tests {
+    if (get_var("ENCRYPT")) {
+        if (get_var('ENCRYPT_ACTIVATE_EXISTING')) {
+            if (get_var('ENCRYPT_FORCE_RECOMPUTE')) {
+                loadtest 'installation/partitioning/lvm_encrypt_recompute_existing';
+            }
+            else {
+                loadtest 'installation/partitioning/lvm_encrypt_should_compute_existing';
```

Comment:
Are you sure this long module name is still displayed nicely?

Reply:
Namings changed to be more clear.